usage/rest-api/src/main/java/brooklyn/rest/domain/EntitySpec.java

```diff
@@ -19,14 +19,19 @@
 package brooklyn.rest.domain;
 
 import static com.google.common.base.Preconditions.checkNotNull;
+
 import com.google.common.collect.ImmutableMap;
+
 import org.codehaus.jackson.annotate.JsonProperty;
 
+import java.io.Serializable;
 import java.util.Collections;
 import java.util.Map;
 
-public class EntitySpec implements HasName {
+public class EntitySpec implements HasName, Serializable {
```

Comment:
Does this restrict the possible values for spec configuration to serializable instances only?